crane/network.go

```diff
@@ -0,0 +1,33 @@
+package crane
+
+import (
+	"fmt"
+	"os"
+)
+
+type Network interface {
+	Name() string
+	Create()
+	Exists() bool
+}
+
+type network struct {
+	RawName string
+}
+
+func (n *network) Name() string {
```

Comment:
Sounds like it should be that way, yes. Unfortunately, implementation is not straight-forward. So maybe we can merge this without it and make that change later?